cmd/pitr/recoverer/recoverer.go

```diff
@@ -136,16 +137,34 @@ func getStartGTIDSet(c BackupS3) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "new storage manager")
 	}
-
-	infoObj, err := s3.GetObject("xtrabackup_info.00000000000000000000") //TODO: work with compressed file
+	listInfo, err := s3.ListObjects("xtrabackup_info")
 	if err != nil {
-		return "", errors.Wrapf(err, "get %s info", prefix)
+		return "", errors.Wrapf(err, "list %s info fies", prefix)
 	}
+	sort.Strings(listInfo)
 
-	lastGTID, err := getLastBackupGTID(infoObj)
-	if err != nil {
-		return "", errors.Wrap(err, "get last backup gtid")
+	var lastGTID string
+	for _, fileName := range listInfo {
+		format := "txt"
+		infoObj, err := s3.GetObject(fileName)
+		if err != nil {
+			return "", errors.Wrapf(err, "get %s info", prefix)
+		}
+
+		if strings.Contains(fileName, "lz4") {
+			format = "lz4"
+		}
+		if strings.Contains(fileName, "qp") {
+			format = "qp"
+		}
+
+		lastGTID, err = getLastBackupGTID(infoObj, format)
+		if err != nil {
+			return "", errors.Wrap(err, "get last backup gtid")
+		}
+		break
```

Comment:
why do we need this break?
Should we process only first file?
If yes, for loop is not needed at all

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/acb9a424f577f3a06fdf388b2fdce0775c2157a9